<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountNewDialog</class>
 <widget class="QDialog" name="AccountNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Account Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frBBControls">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="btCancelBB">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="bbFrameLabel">
        <property name="text">
         <string>New Account</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btSaveBB">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QScrollArea" name="saSettingsUser">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>424</width>
          <height>451</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_28">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="3" column="1">
            <widget class="ExtLineEdit" name="lePassword">
             <property name="echoMode">
              <enum>QLineEdit::PasswordEchoOnEdit</enum>
             </property>
             <property name="placeholderText">
              <string>SIP password here</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Your name:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QCheckBox" name="registerCheckBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ExtLineEdit" name="leUserId">
             <property name="placeholderText">
              <string>SIP username here</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLabel" name="lbProviderLink">
             <property name="text">
              <string>To make and receive calls, you should configure at least one SIP account. If you don't have SIP account, please check this page for recommended &lt;a href=&quot;http://taki.sourceforge.net/serviceproviders.htm&quot;&gt;VoIP service providers&lt;/a&gt;.</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Register:</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="ExtLineEdit" name="leDomain">
             <property name="placeholderText">
              <string>SIP provider host</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>User name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="ExtLineEdit" name="leDisplayName">
             <property name="placeholderText">
              <string>Your name here</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Domain:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frDesktopControls">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btSave">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtLineEdit</class>
   <extends>QLineEdit</extends>
   <header>extlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leDisplayName</tabstop>
  <tabstop>leUserId</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>leDomain</tabstop>
  <tabstop>registerCheckBox</tabstop>
  <tabstop>btSave</tabstop>
  <tabstop>btCancel</tabstop>
  <tabstop>btCancelBB</tabstop>
  <tabstop>btSaveBB</tabstop>
  <tabstop>saSettingsUser</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btCancelBB</sender>
   <signal>clicked()</signal>
   <receiver>AccountNewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSaveBB</sender>
   <signal>clicked()</signal>
   <receiver>AccountNewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>731</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btSave</sender>
   <signal>clicked()</signal>
   <receiver>AccountNewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancel</sender>
   <signal>clicked()</signal>
   <receiver>AccountNewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leDisplayName</sender>
   <signal>returnPressed()</signal>
   <receiver>leUserId</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leUserId</sender>
   <signal>returnPressed()</signal>
   <receiver>leDomain</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTestRing(int)</slot>
 </slots>
</ui>
